### DISCUSSION TITLE
Stand up Monday May 2nd

### DISCUSSION THREAD

**Discussion by CristianCazares** [Bugs-io/TC3001B#81] [Main Stand Ups]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
- 

**What are your priorities for the next 24 hours?**:
- 

**Blockers**:
- 

**Shoutouts**:
- @username for x
```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
- Sent PR with test suite and action on back-end

**What are your priorities for the next 24 hours?**:
- Define which graph are we going to display and start working on their API

**Blockers**:
- None

**Shoutouts**:
- @username for x

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
- Helped @CarlosCRG19 with the backend test suite

**What are your priorities for the next 24 hours?**:
- Worked on graphs 

**Blockers**:
-  None

**Shoutouts**:
- @chivas we going for the 13th cup

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
- Minor enhancemnets on the grid
- Lock and unlock grid interaction

**What are your priorities for the next 24 hours?**:
- Define wether work in testing now, or move into full graphs

**Blockers**:
- Zzz

**Shoutouts**:
- @BrandonMagana bc Chivas won thanks to him

**Comment by Jacobgtz024** [comment]
**What did you achieve in the last 24 hours?**:
- Pushed endpoint for first graph
- Started plotting other graphs

**What are your priorities for the next 24 hours?**:
- Discuss how many graphs we're going to use
- Plot them and turn them into endpoints

**Blockers**:
- None
